LibraBFT/Impl/Consensus/RoundManager/Properties.agda

```diff
@@ -294,7 +294,7 @@ module ensureRoundAndSyncUpMSpec
       -- General invariants / properties
       rmInv         : Preserves RoundManagerInv pre post
       noEpochChange : NoEpochChange pre post
-      noVoteOuts    : OutputProps.NoVotes outs
+      noVoteOuts    : OutputProps.NoMsgs outs
```

Comment:
Even though `ensureRoundAndSyncUpM` does not *currently* produce any messages, it very well may once syncing up is implemented (the peer might send a local message to itself indicating that there is a new epoch). I therefore think we should try to avoid using the fact that this function produces no output messages in proofs.